tests/integration/FtpClientTest.php

```diff
@@ -522,14 +522,17 @@ public function testCopyFromLocalWithFile() : void
         unlink($localFile);
     }
 
+    /**
+     * @depends testCopyToLocalWithDirectory
```

Comment:
The It seems that the `testCopyToLocalWithFile` depends on `testCopyToLocalWithDirectory`.

Once the directory is created on FTP server, it will have the remote directory to access and create the file.

Reply:
I'm not agreed, the `testCopyToLocalWithFile` can run successfully even if the `testCopyToLocalWithDirectory` before, because the logic that handle these two cases are different.

Reply:
I add this raised exception on the line of `1053` inside `FtpClient::copyToLocal` method.